src/main/webapp/WEB-INF/views/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <title>Успей поесть!</title>
+    <link rel="stylesheet" type="text/css" href="../../resources/css/index.css">
```

Comment:
all links to js or css files do like 
<code><link href="<@c.url value="/resources/css/main.css" />" rel="stylesheet" media="screen"/></code>